src/Norm/_getprocnumber/GetProcNumber.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024 Orange. All rights reserved.
+// This software is distributed under the BSD 3-Clause-clear License, the text of which is available
+// at https://spdx.org/licenses/BSD-3-Clause-Clear.html or see the "LICENSE" file for more details.
+
+#include "SystemResource.h"
+#include "CommandLine.h"
+
+int main(int argc, char** argv)
+{
+
+	CommandLine commandLine;
+
+	// Use CommandLine to display global help about the program
+	commandLine.SetCommandName("_getprocnumber");
+	commandLine.SetGlobalHelp(
+	    "This program is used by Khiops. Displays the number of physical cores that are available.");
+	commandLine.ParseMainParameters(argc, argv);
```

Comment:
Do we need to display help for this program, knowing that it is "private" in some sense? Just not displaying this help would make the program even slimmer, as the `CommandLine` header would not be needed anymore, right?

Reply:
I think it is a good thing to respond to the '-h' flag. It doesn't cost much to develop but you are making a point: with CommandLine the size of the binary is 339Ko, without it it is 59Ko... @marcboulle do you have an opinion ?

Reply:
Passer de 339Ko à 59Ko me semble intéressant: c'est loin d'être négligeable.

On peut éviter la classe Command line (over-killing..), et implémenter en quelques lignes de code la petite aide du -h par analyse élémentaire de (argc, argv).